package.json

```diff
@@ -36,7 +36,9 @@
       "pre-push": "npm run test"
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "keytar": "^6.0.1"
```

Comment:
Remove

Reply:
One of the tests fails without this dependency

Reply:
You did not touch anything that would impact this; so if the test fails, it's because you changed something else. You shouldn't add this package.